lib/jekyll/theme.rb

```diff
@@ -6,6 +6,8 @@ class Theme
 
     def initialize(name)
       @name = name.downcase.strip
+      Jekyll.logger.debug "Theme:", name
+      Jekyll.logger.debug "Theme source:", root
       configure_sass
```

Comment:
I added one more line in my local copy to check if path to layouts was as expected in Windows:
```ruby
Jekyll.logger.debug "Layouts Source:", layouts_path
```